### DISCUSSION TITLE
Give a discount only in the backend

### DISCUSSION THREAD

**Discussion by stdt** [liedekef/events-made-easy#666] [Q&A] [answered]
As I understand the discount settings, discount codes can only be entered when booking.

Is it possible to give a discount in the backend after a booking has been made?

So I would like to subsequently reduce the total price for a booking or make the booking completely free.
Is that possible?
If so, how do you do that?

Best regards
StDt

**Comment by liedekef** [comment]
You can change the discount in the backend by entering a fixed discount price in the discount field when editing a booking (it is also explained below the discount field when editing the booking). Or just go ahead and mark a booking as paid, whichever suits the situation best :-)

**Comment by stdt** [comment]
I should have said, that there was no discount field in the booking. 

My template looked similar to this here:
`<table class="eme-rsvp-form">
<tbody>
<tr>
<th scope="row">Vorname*:</th>
<td>#REQ_FIRSTNAME</td>
</tr>
<tr>
<th scope="row">Nachname*:</th>
<td>#_LASTNAME</td>
</tr>
<tr>
<th>Straße*:</th>
<td>#REQ_ADDRESS1</td>
</tr>
<tr>
<th>PLZ*:</th>
<td>#REQ_ZIP</td>
</tr>
<tr>
<th>Ort*:</th>
<td>#REQ_CITY</td>
</tr>
<tr>
<th scope="row">E-Mail*:</th>
<td>#_EMAIL</td>
</tr>
<tr>
<th scope="row">Telefonnummer:</th>
<td>#_PHONE</td>
</tr>
<tr>
<th scope="row"></th>
<td>#_SEATS</td>
</tr>
<tr>
<th scope="row">Buchung als*:</th>
<td>#_FIELD{FT24 Buchungskategorie}</td>
</tr>
<tr>
<th scope="row">Ich bin ...*:</th>
<td>#_FIELD{FT24 Ruhestand}</td>
</tr>
<tr>
<th scope="row">Zimmerauswahl:</th>
<td>#_FIELD{FT24 Zimmerbuchung}</td>
</tr>
<tr>
<th scope="row">Essenswunsch*:</th>
<td>#_FIELD{FT24 Essensauswahl}</td>
</tr>
<tr>
<th scope="row">Kommentar:Z. B. bei Doppelzimmerwahl den Namen der zweiten Person angeben, Angabe von Allergien, ...</th>
<td>#_COMMENT</td>
</tr>
</tbody>
</table>
<p style="text-align: center;"><strong>Gesamtsumme: #_DYNAMICPRICE</strong></p>
<p style="text-align: center;"><strong>#_SUBMIT</strong></p>`

So it seems to me, I cannot add a discount afterwards.

**Comment by liedekef** [comment, marked as answer]
Then there are 2 options:
- in the backend, go to the list of bookings and select in the mass-action dropdown "Partial payment" and indicate how much is already paid (consider that "the discount")
- or add a #_DISCOUNT field to the form, but use [eme_if] around it and check for the value of #_IS_ADMIN_PAGE:
`  [eme_if tag="#_IS_ADMIN_PAGE" value="1"]Enter a discount: #_DISCOUNT[/eme_if]  
`  That will cause the discount field to only be shown in the admin backend (well, at least it should :-) ).

**Reply by stdt** [reply]
`eme_if` works like a charm! 

That is exactly what I wanted to do!
Thank you very much!
